inc/Plugin.php

```diff
@@ -0,0 +1,256 @@
+				}
+			} else {
+				$markup .= ' class="medium" disabled>';
+			}
+			$markup .= '</select></div>';
+			return $markup;
+		}
+		return $input;
+	}
+
+	/**
+	 * Settings fields for this field type.
+	 * @action gform_field_standard_settings
+	 */
+	static function settings( $position, $form_id ) {
+		if ( $position == 25 ) {
+		?>
+			<li class="term_select_settings field_setting">
+				<label for="term_select_tax">
+					Taxonomy
+					<select id="term_select_tax" onchange="SetFieldProperty('termSelectTax', jQuery(this).val());">
+						<option> - select - </option>
+					<?php
+					// list out all of the taxonomies
+					foreach ( get_taxonomies( [], 'objects' ) as $name => $tax ) {
+						echo '<option value="' . esc_attr( $name ) . '">' . esc_html( $tax->label ) . '</option>';
+					}
+					?>
+					</select>
+				</label>
+				<?php //gform_tooltip( 'geo_field' ); ?>
+				<label for="term_select_max">
+					Maximum Selected Terms
+					<input type="text" id="term_select_max" onkeyup="SetFieldProperty('termSelectMax', this.value);" size="4">
+				</label>
+			</li>
+		<?php
+		}
+	}
+
+	/**
+	 * @filter gform_tooltips
+	 */
+	static function add_tooltips( $tooltips ) {
+		//$tooltips['geo_field'] = '<h6>Places Field</h6>Enter a field that would be returned from a Google Places result to be populated upon selection of a result.';
+		return $tooltips;
+	}
+
+	/**
+	 * Fill in defaults for a new field
+	 * @action gform_editor_js_set_default_values
+	 */
+	static function field_edit_defaults() {
+	?>
+		case <?php echo json_encode( self::type ); ?>:
+			field.label = 'Terms';
+		break;
+	<?php
+	}
+
+	/**
+	 * Additional JS to be included after gravity forms JS
+	 * @action gform_editor_js
+	 */
+	static function editor_js() {
+	?>
+		<script type="text/javascript">
+			//defining settings for the new custom field
+			fieldSettings[<?php echo json_encode( self::type ); ?>] = '.term_select_settings, .conditional_logic_field_setting, .error_message_setting, .label_setting, .label_placement_setting, .rules_setting, .admin_label_setting, .size_setting, .visibility_setting, .placeholder_setting, .description_setting, .css_class_setting';
+
+			(function($) {
+				//binding to the load field settings event to initialize the field
+				$(document).bind('gform_load_field_settings', function(event, field, form){
+					$("#term_select_tax").val(field['termSelectTax']);
+					$("#term_select_max").val(field['termSelectMax']);
+				});
+			})(jQuery);
+		</script>
+	<?php
+	}
+
+	/**
+	 * Enqueue scripts needed for the geo complete field
+	 * @param  array $form form properties
+	 * @param  bool $ajax ajax form or not
+	 * @action gform_enqueue_scripts
+	 */
+	static function enqueue_scripts( $form, $ajax ) {
+		foreach ( $form['fields'] as $field ) {
+			if ( self::field_is_this( $field ) ) {
+				wp_enqueue_script( 'chosen-js-term-select', PLUGINS_URL . '/assets/chosen.jquery.min.js', [ 'jquery' ] );
```

Comment:
Not editing it (for now) - but I wanted to make sure that we had the right version, since if `chosen-js` was already registered previously with a different version then it doesn't get re-registered. Meanwhile, chosen may already be bundled with g-forms and we could maybe just use that. I'm just going to open an issue on this repo as a reminder.